### ISSUE TITLE
Incorrect initial states

### ISSUE THREAD

**Issue by gvermillion** [simfection/simfection#1]
# Overview

Default `initial_infected:0.2` should produce, on average, as population with 20% of agents infected and the remaining susceptible. 

Actual initial infected rated are 90%+...need to figure out why. 

It is likely related to [`simfection/population_engine.py:PopulationEngine._synthesize_states`](https://github.com/gvermillion/simfection/blob/master/simfection/population_engine.py#L25-L47).

# To Do

- Test `PopulationEngine._synthesize_states` on aggregate
- Update method if bad results
- Make unit test for method. Make it so the initial states have to be reproduced on average within a certain tolerance for a given number of runs.

**Comment by alexanderogle** [comment]
The method `PopulationEngine._synthesize_states()` is never called within the class. It is only defined. This is probably why changing the default settings for initial states doesn't seem to affect the synthesized states. I wrote some initial testing in feature/test_population_engine. I set `initial_states = {'inf': 0.15, 'dead': 0.2}` and see tests failing because the actual percentage of states for 'dead' is 0.0 instead of 0.2 (the states aren't being synthesized).

**Comment by gvermillion** [comment]
You're right. I never actually implemented `_synthesize_states` after I wrote it. I corrected it.

To test it, I build a notebook that does the initial synthesis `N` times and then compares the initial states to the expectation. I'm satisfied it's working now. If you pull from `develop`, you'll find the notebook [here](https://github.com/simfection/simfection/blob/develop/notebooks/issues/incorrect_initial_states.ipynb).

Also, I rolled back your merges to develop, at least the ones that I could. You were correct in creating a branch for the work on the issue, but don't merge the branch back into `develop` until the issue is resolved. We want `develop` to be as clean a codebase as possible.

That said, I went ahead and merged my fix for this issue into `develop`. Once you take a look and are satisfied that it works (play around with the notebook), you can merge it into `master`.

**Comment by gvermillion** [comment]
I see that your merge into `develop` did include a unit test, so that’s still there. 

I closed this even though the unit test may not be ready/right yet. I don’t really understand unit tests yet, so maybe you can explain to me/the group their use and motivation for doing so. 

In any case, I don’t want to leave issues open, so we can create another isssue for the unit test.
